extensions/applicationinsights-angularplugin-js/package.json

```diff
@@ -1,48 +1,50 @@
 {
-    "name": "@microsoft/applicationinsights-angularplugin-js",
+  "name": "@microsoft/applicationinsights-angularplugin-js",
     "version": "2.5.10",
     "description": "Microsoft Application Insights Angular plugin",
-    "main": "dist/applicationinsights-angularplugin-js.js",
-    "module": "dist-esm/applicationinsights-angularplugin-js.js",
-    "types": "types/applicationinsights-angularplugin-js.d.ts",
-    "sideEffects": false,
     "repository": {
         "type": "git",
         "url": "https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/applicationinsights-angularplugin-js"
     },
-    "scripts": {
-        "prepublishOnly": "npm install && npm run build",
-        "build": "npm run build:esm && npm run build:browser",
-        "build:esm": "tsc -p tsconfig.json",
-        "build:browser": "rollup -c",
-        "test": "cd Tests/test-app && npm install && ng test",
-        "lint": "tslint -p tsconfig.json"
-    },
-    "devDependencies": {
-        "@microsoft/applicationinsights-rollup-es3": "1.1.3",
-        "@microsoft/applicationinsights-properties-js": "2.5.10",
-        "@angular/core": "~8.1.2",
-        "@angular/router": "~8.1.2",
-        "grunt": "1.0.1",
-        "rollup": "^0.66.0",
-        "rollup-plugin-commonjs": "^9.3.4",
-        "rollup-plugin-node-resolve": "^3.4.0",
-        "rollup-plugin-replace": "^2.1.0",
-        "rollup-plugin-uglify": "^6.0.0",
-        "ts-node": "^8.3.0",
-        "typescript": "~3.4.3",
-        "rxjs": "~6.4.0",
-        "types": "^0.1.1",
-        "zone.js": "~0.9.1",
-        "@angular/common": "~8.1.2",
-        "@angular/platform-browser": "~8.1.2",
-        "tslint": "^5.19.0",
-        "tslint-config-prettier": "^1.18.0"
-    },
-    "dependencies": {
-        "@microsoft/applicationinsights-shims": "^1.0.3",
-        "@microsoft/applicationinsights-common": "2.5.10",
-        "@microsoft/applicationinsights-core-js": "2.5.10"
-    },
-    "license": "MIT"
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~11.0.6",
+    "@angular/common": "~11.0.6",
+    "@angular/compiler": "~11.0.6",
+    "@angular/core": "~11.0.6",
+    "@angular/forms": "~11.0.6",
+    "@angular/platform-browser": "~11.0.6",
+    "@angular/platform-browser-dynamic": "~11.0.6",
+    "@angular/router": "~11.0.6",
+    "rxjs": "~6.6.0",
+    "tslib": "^2.0.0",
+    "zone.js": "~0.10.2"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.1100.6",
```

Comment:
You need to include the applicationinsights packages based  on the build errors

Cannot find module '@microsoft/applicationinsights-common'
Cannot find module '@microsoft/applicationinsights-core-js'
Cannot find module '@microsoft/applicationinsights-properties-js'